components/camel-rest-swagger/src/main/java/org/apache/camel/component/rest/swagger/RestSwaggerEndpoint.java

```diff
@@ -592,31 +623,54 @@ static String hostFrom(final RestConfiguration restConfiguration) {
      * @return the specification
      * @throws IOException
      */
-    static Swagger loadSpecificationFrom(final CamelContext camelContext, final URI uri) throws IOException {
+    static Swagger loadSpecificationFrom(final CamelContext camelContext, final URI uri, SSLContextParameters sslContextParameters) throws IOException {
         final ObjectMapper mapper = Json.mapper();
 
         final SwaggerParser swaggerParser = new SwaggerParser();
 
         final String uriAsString = uri.toString();
 
-        try (InputStream stream = ResourceHelper.resolveMandatoryResourceAsInputStream(camelContext, uriAsString)) {
-            final JsonNode node = mapper.readTree(stream);
+        if (sslContextParameters == null) {
+            try (InputStream stream = ResourceHelper.resolveMandatoryResourceAsInputStream(camelContext, uriAsString)) {
+                return parseInputStream(swaggerParser, mapper, stream);
+            } catch (final Exception e) {
+                return loadSpecificationFalback(swaggerParser, uriAsString, e);
```

Comment:
I think you meant 
```suggestion
                return loadSpecificationFallback(swaggerParser, uriAsString, e);
```

Reply:
I've fixed this typo in last force push, but evidently there was one missing case, I'll fix it.